src/xunit.runner.utility/Project/XunitFilters.cs

```diff
@@ -43,6 +47,43 @@ public XunitFilters()
         public HashSet<string> IncludedMethods { get; }
 
         /// <summary>
+        /// 
```

Comment:
Thanks for catching that. I'll fix.
On Sun, Feb 7, 2016 at 11:17 PM Robert Jørgensgaard Engdahl <
notifications@github.com> wrote:

> In src/xunit.runner.utility/Project/XunitFilters.cs
> <https://github.com/xunit/xunit/pull/765#discussion_r52137939>:
>
> > @@ -43,6 +47,43 @@ public XunitFilters()
> >          public HashSet<string> IncludedMethods { get; }
> >
> >          /// <summary>
> > +        ///
>
> you forgot this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xunit/xunit/pull/765/files#r52137939>.
>